### ISSUE TITLE
'caporal' dependency not found at runtime when running the cli

### ISSUE THREAD

**Issue by yusefnapora** [pact-foundation/pact-node#72]
When installing pact via `yarn add --dev @pact-foundation/pact-node`, trying to run the cli from `./node_modules/.bin` produces the following error:

```
./node_modules/.bin/pact stub
module.js:557
    throw err;
    ^

Error: Cannot find module 'caporal'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/yusef/work/code/yusef/s4l-catalog-api-client/node_modules/@pact-foundation/pact-node/bin/pact-cli.js:5:11)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
```

This seems to be because the `caporal` module is listed in `devDependencies` in package.json.  Since it's required at runtime, it seems like this should be moved to the `dependencies` section.

**Comment by mboudreau** [comment]
Crap, yes, you're right, let me fix that right now.

**Comment by mboudreau** [comment]
Fixed in release 6.7.4. Cheers.
